lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1014,6 +1018,17 @@ SPIRVModuleImpl::addPointerType(SPIRVStorageClassKind StorageClass,
   return addType(Ty);
 }
 
+SPIRVTypeUntypedPointerKHR *
+SPIRVModuleImpl::addUntypedPointerKHRType(SPIRVStorageClassKind StorageClass) {
+  auto Loc = UntypedPtrTyMap.find(StorageClass);
```

Comment:
Post review comment: apparently it's a minor bug inherited from addPointerType method, that was changed by me in f0d7e4769 . See: 2.8. Types and Variables:
"Pointer types are also allowed to have multiple <id>s for the same opcode and
operands, to allow for differing decorations (e.g., Volatile) or different decoration values (e.g., different
Array Stride values for the ArrayStride)."
Like this we are preventing a case, when we need to create multiple decorated with different decorations pointer types. Will create an issue for that.